tests/sandbox/message_batches/create_message_batches/test_success.py

```diff
@@ -69,31 +80,41 @@ def test_201_message_batch_valid_nhs_number(nhsd_apim_proxy_url):
     .. include:: ../../partials/happy_path/test_201_message_batch_valid_nhs_number.rst
     """
     data = Generators.generate_valid_create_message_batch_body("sandbox")
-    data["data"]["attributes"]["messages"][0]["recipient"]["nhsNumber"] = constants.VALID_NHS_NUMBER
+    data["data"]["attributes"]["messages"][0]["recipient"][
+        "nhsNumber"
+    ] = constants.VALID_NHS_NUMBER
 
-    resp = requests.post(f"{nhsd_apim_proxy_url}{MESSAGE_BATCHES_ENDPOINT}", headers={
-            "Accept": "application/json",
-            "Content-Type": "application/json"
-        }, json=data
+    resp = requests.post(
+        f"{nhsd_apim_proxy_url}{MESSAGE_BATCHES_ENDPOINT}",
+        headers={"Accept": "application/json", "Content-Type": "application/json"},
+        json=data,
+    )
+    Assertions.assert_201_response(
+        resp,
+        data["data"]["attributes"]["messageBatchReference"],
+        data["data"]["attributes"]["routingPlanId"],
     )
-    Assertions.assert_201_response(resp, data["data"]["attributes"]["messageBatchReference"])
 
 
 @pytest.mark.sandboxtest
-@pytest.mark.parametrize('dob', constants.VALID_DOB)
+@pytest.mark.parametrize("dob", constants.VALID_DOB)
 def test_201_message_batch_valid_dob(nhsd_apim_proxy_url, dob):
     """
     .. include:: ../../partials/happy_path/test_201_message_batch_valid_dob.rst
     """
     data = Generators.generate_valid_create_message_batch_body("sandbox")
     data["data"]["attributes"]["messages"][0]["recipient"]["dateOfBirth"] = dob
 
-    resp = requests.post(f"{nhsd_apim_proxy_url}{MESSAGE_BATCHES_ENDPOINT}", headers={
-            "Accept": "application/json",
-            "Content-Type": "application/json"
-        }, json=data
+    resp = requests.post(
+        f"{nhsd_apim_proxy_url}{MESSAGE_BATCHES_ENDPOINT}",
+        headers={"Accept": "application/json", "Content-Type": "application/json"},
+        json=data,
+    )
+    Assertions.assert_201_response(
+        resp,
+        data["data"]["attributes"]["messageBatchReference"],
```

Comment:
Non blocking.

Might be worth moving these into variables, its quite a horrible thing to read

Reply:
I don't mind it. I think it make it clear the value is coming from the input data and keeps it simple.